src/MountedTraversal.js

```diff
@@ -177,6 +178,12 @@ export function parentsOfInst(inst, root) {
   return pathToNode(inst, root).reverse();
 }
 
+export function instMatchesObjectProps(inst, props) {
+  if (!isDOMComponent(inst)) return false;
+  const node = getNode(inst);
+  return !isEmpty(props) && isMatch(propsOfNode(node), props);
```

Comment:
That is true. But how often is somebody going to look for something with an empty object? That just seems like a design against the wrong use-case. And I guess maybe somebody does want to search for nodes that don't have any props? which `find({})` could support? feels awkward though. I guess I could see either way here.